### PR TITLE
PREEMPT_RT latency performance for SDK 10.1

### PR DESCRIPTION
**Pull request by bryanbrattlof**
Hello Everyone!

This small series updates the PREEMPT_RT latency performance numbers for the CoreSDK 10.1 release observed when using a simple CPU based background load.

Happy Hacking
~Bryan

### CODE REVIEW

**Review by praneethbajjuri — DISMISSED** [review]

**Comment by praneethbajjuri** [comment]
> New warnings found with rstcheck:
> ```text
> source/devices/AM64X/linux/RT_Linux_Performance_Guide.rst:502: (INFO/1) Duplicate implicit target name: "am64xx-evm".
> ```
> 

Address this warning

**Review by StaticRocket — CHANGES_REQUESTED** [review]
Content is fine. Just style issues.

File: source/devices/AM62AX/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (cshilwant) [inline source/devices/AM62AX/linux/img/rt-cpu-method-all-latency-histogram.png:1]: Any particular reasons for creating a new img directory, instead reusing https://github.com/TexasInstruments/processor-sdk-doc/tree/master/source/images

File: source/devices/AM62PX/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (cshilwant) [inline source/devices/AM62PX/linux/img/rt-cpu-method-all-latency-histogram.png:1]: Same comment as above

File: source/devices/AM62X/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (cshilwant) [inline source/devices/AM62X/linux/img/rt-cpu-method-all-latency-histogram.png:1]: Same comment as above

File: source/devices/AM64X/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (cshilwant) [inline source/devices/AM64X/linux/img/rt-cpu-method-all-latency-histogram.png:1]: Same comment as above

**Review by cshilwant — DISMISSED** [review]

File: source/devices/AM62AX/linux/RT_Linux_Performance_Guide.rst [hunk 1]
```diff
@@ -49,15 +49,21 @@ It is one of the most frequently used tools for evaluating the relative performa
...
 
-Test command for running stress-ng and cyclictest together
+Test commands used for running stress-ng and cyclictest together
 
-``stress-ng --cpu-method=all -c 4 &``
+.. code:: console
 
-``cyclictest -m -Sp98 -D6h -h400 -i200 -q``
+    stress-ng --cpu-method=all -c 4 &
+    cyclictest -m -Sp80 -D6h -h400 -i200 -M -q
```
[the remark this answers is not in the document]
Comment (bryanbrattlof) [reply source/devices/AM62AX/linux/RT_Linux_Performance_Guide.rst:57]: ha I'll learn how to count one day

File: source/devices/AM62AX/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (bryanbrattlof) [reply source/devices/AM62AX/linux/img/rt-cpu-method-all-latency-histogram.png:1]: all of these images will be different for every platform. why would we want them in a general folder?

File: source/devices/AM62PX/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (bryanbrattlof) [reply source/devices/AM62PX/linux/img/rt-cpu-method-all-latency-histogram.png:1]: same as above

File: source/devices/AM62X/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (bryanbrattlof) [reply source/devices/AM62X/linux/img/rt-cpu-method-all-latency-histogram.png:1]: same as above

File: source/devices/AM64X/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (bryanbrattlof) [reply source/devices/AM64X/linux/img/rt-cpu-method-all-latency-histogram.png:1]: same as above

File: source/devices/AM64X/linux/RT_Linux_Performance_Guide.rst [hunk 2]
```diff
@@ -493,9 +495,6 @@ AM64xx-EVM
 OSPI Flash Driver
 -------------------------
 
-AM64XX-EVM
```
Comment (StaticRocket) [inline source/devices/AM64X/linux/RT_Linux_Performance_Guide.rst:496]: This change causes a (valid) section depth jump error.

**Review by StaticRocket — CHANGES_REQUESTED** [review]

File: source/devices/AM62AX/linux/img/rt-cpu-method-all-latency-histogram.png
Comment (cshilwant) [reply source/devices/AM62AX/linux/img/rt-cpu-method-all-latency-histogram.png:1]: @bryanbrattlof All images irrespective of device specific/ generic are all maintained under `source/images` and I have seen folks using SoC names as differentiator (which you will see it under source/images as well)

**Review by praneethbajjuri — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Comment by cshilwant** [comment]
@StaticRocket / @DhruvaG2000 / @AashvijShenai for final review

**Review by StaticRocket — APPROVED** [review]
As much as I'd like to standardize asset locations, we have enough measures in place to prevent issues with people putting images and files near their RST source files. This is fine.

**Pull request merged**